decidim-meetings/app/views/decidim/meetings/meetings/_meetings.html.erb

```diff
@@ -1,5 +1,17 @@
+  <% if @meetings_warning %>
```

Comment:
I'd simplify this, if `meetings.length == 0` just show a message telling the user that there are no meetings, no need to have two different warnings.